ouroboros-consensus-cardano/src/Ouroboros/Consensus/Cardano/Block.hs

```diff
@@ -210,7 +222,18 @@ pattern BlockMary b = HardForkBlock (OneEraBlock (TagMary (I b)))
 pattern BlockAlonzo :: ShelleyBlock (TPraos c) (AlonzoEra c) -> CardanoBlock c
 pattern BlockAlonzo b = HardForkBlock (OneEraBlock (TagAlonzo (I b)))
 
-{-# COMPLETE BlockByron, BlockShelley, BlockAllegra, BlockMary, BlockAlonzo #-}
+pattern BlockBabbage :: ShelleyBlock (Praos c) (BabbageEra c) -> CardanoBlock c
+pattern BlockBabbage b = HardForkBlock (OneEraBlock (TagBabbage (I b)))
+
+{-# COMPLETE
```

Comment:
Style: let's use leading `,`s, so that just have to add one line each time (and match the existing COMPLETE pragmas in this module)